### PR TITLE
[MU4] Disabled use std filesystem (fix compiling with MinGW)

### PR DESCRIPTION
**Pull request by igorkorsukov**
Disabled use std filesystem (fix compiling with MinGW)

### CODE REVIEW

File: thirdparty/haw_logger/logger/logdefdest.cpp [hunk 1]
```diff
@@ -7,15 +7,16 @@
...
+//    #if __has_include(<filesystem>)
+//        #include <filesystem>
+//        namespace fs = std::filesystem;
+//    #elif __has_include(<experimental/filesystem>)
+//        #include <experimental/filesystem>
+//        namespace fs = std::experimental::filesystem;
+//    #else
+//        #error compiler must either support c++17
+//    #endif
+#error At moment supported only with Qt
```
Comment (Jojo-Schmitz) [inline thirdparty/haw_logger/logger/logdefdest.cpp:19]: With Qt?

**Comment by igorkorsukov** [comment]
Known Issues

* MacOS 10.15 XCode 10.3 on CI not has `<filesystem>`
* MacOS 10.14 has `<filesystem>` (pass check `#if __has_include(<filesystem>)`) but doesn't have implementation (path, exists) - very strange 
* gcc 7 (on CI, default for ubuntu 18.10) has `<experimental/filesystem>` and needs link `stdc++fs` lib (work fine)
* mingw has `<filesystem>`, but doesn't have `stdc++fs` lib

see also https://stackoverflow.com/questions/54290254/problem-adding-stdfilesystem-to-cmake-project

File: thirdparty/haw_logger/logger/logdefdest.cpp (hunk 1 above)
Comment (igorkorsukov) [reply thirdparty/haw_logger/logger/logdefdest.cpp:19]: yes, at the moment using QDir   
(The original idea was not to depend on Qt)

Comment (igorkorsukov) [reply thirdparty/haw_logger/logger/logdefdest.cpp:19]: needs to define HAW_LOGGER_QT_SUPPORT and have QDir 
(we have it)

Comment (Jojo-Schmitz) [reply thirdparty/haw_logger/logger/logdefdest.cpp:19]: Ah, I see (now)

**Pull request merged**